cypress/e2e/group1/checkerWorkflowFromTools.ts

```diff
@@ -35,6 +35,7 @@ describe('Checker workflow test from my-tools', () => {
   describe('Should be able to register and publish a checker workflow from a tool', () => {
     it('visit a tool and have the correct buttons and be able to register a checker workflow', () => {
       cy.intercept('api/containers/*?include=validations').as('getTool');
+      cy.visit('/my-tools');
```

Comment:
Hm, I tried removing the extra visit in this [commit](https://github.com/dockstore/dockstore-ui2/commit/4da6c180a56b6b9c3098aa2ce6ee6ae9156d6408), looks like [the test didn't flake](https://cloud.cypress.io/projects/1ya4vj/runs/33847/overview/9f749865-b90d-4fd5-98f1-6319ed317e8c/replay?roarHideRunsWithDiffGroupsAndTags=1&runStatus=RUNNING&ts=1714066807055.3&att=1). Not too sure why the previous errors occurred... (might not have been a direct cause and effect)

I think the single visit is more logical, so I'll keep that for now? But we can keep an eye on it for future flakiness.

Reply:
Yeah, I think we can leave the single visit for now :slightly_smiling_face:

Reply:
Hmm, it flaked again here :sob: https://cloud.cypress.io/projects/1ya4vj/runs/33852/overview/4559a516-5f43-452c-bbd5-5f852cd35cb5?roarHideRunsWithDiffGroupsAndTags=1&runStatus=RUNNING

Reply:
>  (not sure if this applies to us? https://docs.cypress.io/api/commands/visit#Prevent-requests-before-a-remote-page-initially-loads)

Hmm, it may be worth a try to put the intercept in `beforeEach` before the `cy.visit` line like the link suggests.

If it doesn't work then you can leave it the way you have in the PR, but maybe add a comment about why the extra visit is there